index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+                                                  <header>
+                                                    <h1 id="logo">Aziz Mejri</h1>
+                                                    <button type="button" class="menu-bars" id="menu_btn">
+                                                      <i class="fas fa-bars"></i>
+                                                    </button>
+                                                    <nav id="desktop-nav">
+                                                      <ul>
+                                                        <li><a class="nav-link" href="#projects">Portoflio</a></li>
+                                                        <li><a class="nav-link" href="#about-me">About</a></li>
+                                                        <li>
+                                                          <a class="nav-link" href="#contact"
+                                                            >Contact <i class="far fa-envelope"></i
+                                                          ></a>
+                                                        </li>
+                                                      </ul>
+                                                    </nav>
+                                                    <nav id="mobile-nav">
+      <span>x</span>
+      <ul>
+        <li><a class="nav-link" href="#projects">Portfolio</a></li>
+        <li><a class="nav-link" href="#about-me">About</a></li>
+        <li>
+          <a class="nav-link" href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] You did a great work to match the design like in [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as the previous reviewer mentioned, there is some minor change to match the design. The `nav-link` items should be aligned to the left, not the center to be like the Figma design because now you learn to become a professional developer you have to be able to follow project requirements strictly like the developers how to build actual projects in companies.

| Your design | Figma template design | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/53402600/169526642-f2b2f508-133a-4bed-a3cf-b4d549ebab8d.png) | ![image](https://user-images.githubusercontent.com/53402600/169526675-d86d6ea8-83ad-4265-9743-984ce68898eb.png) |